tests/v2/validation/rbac/rbac_additional_test.go

```diff
@@ -60,116 +52,131 @@ func (rb *RBACAdditionalTestSuite) SetupSuite() {
 	require.NoError(rb.T(), err)
 }
 
-func (rb *RBACAdditionalTestSuite) ValidateAddStdUserAsProjectOwner() {
+func (rb *RBACAdditionalTestSuite) TestRbacP1ClusterOwner() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
 
-	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+	clusterAdmin, standardUserClient, err := setupUser(rb.client, standardUser)
 	require.NoError(rb.T(), err)
-	rb.standardUserCOProject = createProjectAsCO
 
-	log.Info("Validating if cluster owner can add a user as project owner in a project")
-	err = users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleProjectOwner, nil)
+	rb.T().Logf("Creating an additional standard user")
+	additionalUser, additionalUserClient, err := setupUser(rb.client, standardUser)
 	require.NoError(rb.T(), err)
-	userGetProject, err := projects.GetProjectList(rb.additionalUserClient, rb.cluster.ID)
+
+	rb.T().Logf("Adding user as " + roleOwner + " to the downstream cluster.")
+	err = users.AddClusterRoleToUser(rb.client, rb.cluster, clusterAdmin, roleOwner, nil)
+	require.NoError(rb.T(), err)
+	standardUserClient, err = standardUserClient.ReLogin()
 	require.NoError(rb.T(), err)
-	assert.Equal(rb.T(), 1, len(userGetProject.Data))
-	assert.Equal(rb.T(), rb.standardUserCOProject.Name, userGetProject.Data[0].Name)
 
-	err = users.RemoveProjectMember(rb.standardUserClient, rb.additionalUser)
+	rb.T().Logf("Validating if " + roleOwner + " can add another standard user as a project owner")
```

Comment:
should all these 'validating if..' be broken out into another test? they don't seem to match this test function's purpose based on the name `RbacP1ClusterOwner`.

if this is supporting the test and must be done in this order, can you update the test name to be more clear? I still feel like this should be pulled out into a helper function to make the code more modular / readable.

Reply:
It was previously broken into multiple tests, but these tests are all run as a cluster owner. So, I grouped the tests that are run for a particular cluster role and hence named it RbacP1ClusterOwner >> P1 tests for a cluster owner. any suggestions on what can this be renamed to?

Reply:
I thought we agreed, if they don't need to be ran in a given order to keep them separate. Maybe you could break these out into different files i.e. `clusterOwnerRBACTests.go` and keep the tests separate.